crates/hc_sandbox/src/calls.rs

```diff
@@ -393,64 +393,78 @@ async fn call_inner(client: &mut AdminWebsocket, call: AdminRequestCli) -> anyho
         AdminRequestCli::ListAppWs => {
             let interface_infos = client.list_app_interfaces().await?;
-            msg!("Attached app interfaces {:?}", interface_infos);
+            println!("{}", serde_json::to_value(&interface_infos)?);
         }
         AdminRequestCli::RegisterDna(args) => {
-            let dnas = register_dna(client, args).await?;
-            msg!("Registered DNA: {:?}", dnas);
+            let dna = register_dna(client, args).await?;
+            println!("{}", serde_json::to_value(dna.to_string())?);
         }
         AdminRequestCli::InstallApp(args) => {
             let app = install_app_bundle(client, args).await?;
-            msg!("Installed app: {}", app.installed_app_id,);
+            println!("{}",  convert_app_info(app)?);
         }
         AdminRequestCli::UninstallApp(args) => {
             client
                 .uninstall_app(args.app_id.clone(), args.force)
                 .await?;
-            msg!("Uninstalled app: {}", args.app_id,);
+            msg!("Uninstalled app: \"{}\"", args.app_id);
         }
         AdminRequestCli::ListDnas => {
-            let dnas = client.list_dnas().await?;
-            msg!("DNAs: {:?}", dnas);
+            let dnas: Vec<DnaHashB64> = client.list_dnas()
+                .await?
+                .into_iter()
+                .map(|d| d.into())
+                .collect();
+            println!("{}", serde_json::to_value(&dnas)?);
         }
         AdminRequestCli::NewAgent => {
             let agent = client.generate_agent_pub_key().await?;
-            msg!("Added agent {}", agent);
+            println!("{}", serde_json::to_value(agent.to_string())?);
         }
         AdminRequestCli::ListCells => {
-            let cells = client.list_cell_ids().await?;
-            msg!("Cell IDs: {:?}", cells);
+            let cell_id_values: Vec<serde_json::Value> = client.list_cell_ids()
+                .await?
+                .into_iter()
+                .map(|cell_id| serde_json::to_value(cell_id))
+                .collect::<Result<Vec<serde_json::Value>, serde_json::Error>>()?;
+            let converted = cell_id_values.into_iter()
+                .map(|value| transform_cell_id(value))
+                .collect::<anyhow::Result<Vec<serde_json::Value>>>()?;
+            println!("{}", serde_json::to_value(&converted)?);
         }
         AdminRequestCli::ListApps(args) => {
             let apps = client.list_apps(args.status).await?;
-            msg!("List apps: {:?}", apps);
+            let values = apps.into_iter()
+                .map(|app_info| convert_app_info(app_info))
+                .collect::<anyhow::Result<Vec<serde_json::Value>>>()?;
+            println!("{}", serde_json::to_value(values)?);
         }
         AdminRequestCli::EnableApp(args) => {
             client.enable_app(args.app_id.clone()).await?;
-            msg!("Activated app: {:?}", args.app_id);
+            msg!("Enabled app: \"{}\"", args.app_id);
```

Comment:
Changed wording from `Activated` to `Enabled`, as to be consistent with API names (e.g. `Enable App`)